meta/8cda00f3.csv

```diff
@@ -945,3 +945,37 @@ Id,FileID,Domain,RepoName,FilePath,LineStart:LineEnd,GroundTruth,WithWords,Value
 131113,387aa4d0,GitHub,8cda00f3,data/8cda00f3/src/gz.c,139451:139451,F,F,,,F,F,,,,,0.00,,F,F,F,
 131114,387aa4d0,GitHub,8cda00f3,data/8cda00f3/src/gz.c,139408:139408,F,F,,,F,F,,,,,0.00,,F,F,F,
 131115,3569d385,GitHub,8cda00f3,data/8cda00f3/src/fn.scss,194:194,Template,F,0,0,F,F,NumberOnly,,,Secret,0.00,0,F,F,F,Password
+131861,218dc847,GitHub,8cda00f3,data/8cda00f3/other/bb.md,100:100,T,F,,,F,F,,,,,0.00,,F,F,F,Other
+131862,218dc847,GitHub,8cda00f3,data/8cda00f3/other/bb.md,132:132,T,F,,,F,F,,,,,0.00,,F,F,F,Other
+131863,218dc847,GitHub,8cda00f3,data/8cda00f3/other/bb.md,42:42,T,F,,,F,F,,,,,0.00,,F,F,F,Other
+131864,218dc847,GitHub,8cda00f3,data/8cda00f3/other/bb.md,67:67,T,F,,,F,F,,,,,0.00,,F,F,F,Other
```

Comment:
If it is about VIN rule, then what about set category as Predefined Pattern?

Reply:
No. VIN was added here https://github.com/Samsung/CredData/pull/37/commits/509ff5b8a4adb854ee3447a25b9b1b9a3a28c637
Others data - markup for lost lines

Reply:
Okay.
If `7738e44d.csv` file is about VIN rule, do you think the category of VIN rule is `Others`?

Reply:
***Decided to just use `Others` by messenger.***